src/backend/distributed/utils/resource_lock.c

```diff
@@ -752,14 +752,14 @@ lock_relation_if_exists(PG_FUNCTION_ARGS)
 		PG_RETURN_BOOL(false);
 	}
 
-    /* get the lock mode */
+	/* get the lock mode */
 	lockMode = LockModeTextToLockMode(lockModeCString);
 
-    /* resolve relationId from passed in schema and relation name */
+	/* resolve relationId from passed in schema and relation name */
 	relationNameList = textToQualifiedNameList(relationName);
 	relation = makeRangeVarFromNameList(relationNameList);
 
-    /* lock the relation with the lock mode */
+	/* lock the relation with the lock mode */
```

Comment:
are these just citus_indent changes? could we do them as a separate commit?

Reply:
@velioglu Maybe double check your uncrustify version. We had similar issues last week. I think travis uses `uncrustify 0.63`